torch/distributed/optim/zero_redundancy_optimizer.pyi

```diff
@@ -19,9 +19,9 @@ class _DDPBucketAssignment:
     tensor: torch.Tensor | None
 
 class _OverlapStatus(enum.IntEnum):
-    UNINITIALIZED: int = ...
-    DDP_HAS_REBUILT_BUCKETS: int = ...
-    INITIALIZED: int = ...
+    UNINITIALIZED = ...
```

Comment:
Okay, mypy updated the documentation. If we use ... or know the type but not the value  we should use typing.cast instead https://github.com/python/mypy/blob/master/CHANGELOG.md#change-to-enum-membership-semantics

Reply:
This might be fine though because it's an IntEnum... hmm...

Reply:
That documentation shares instructions for generic enums, where the type isn't known.  This one is an IntEnum, so mypy can presumably infer the type from that.

The evidence: Mypy didn't raise an error on this file